app/main.py

```diff
@@ -1,3 +1,14 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+
+
+def cache(func: Callable) -> Callable:
+    list_of_args = {}
```

Comment:
Use another naming for the `list_of_args` variable, because you store a dict here, not list as naming indicates. You could use `inner_cache` for example.